Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/ReportBugRow.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ReportBugRow.swift
+//  Hedvig
+//
+//  Created by Gustaf Gunér on 2019-02-14.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Foundation
+import Presentation
+import MessageUI
+import DeviceKit
+
+struct ReportBugRow {
+    let presentingViewController: UIViewController
+}
+
+extension ReportBugRow: Viewable {
+    func materialize(events: SelectableViewableEvents) -> (KeyValueRow, Disposable) {
+        let bag = DisposeBag()
+        
+        let row = KeyValueRow()
+                
+        let emailAddress = "ios@hedvig.com"
+        
+        row.keySignal.value = "Rapportera bugg"
+        row.valueSignal.value = emailAddress
+        
+        row.valueStyleSignal.value = .rowTitlePurple
+    
+        let device = Device()
+    
+        let deviceInfo = "Device: \(device)\nSystem: \(device.systemName) \(device.systemVersion)"
```

Comment:
Can we add app-version and member id also?

Reply:
Yes!